apps/prairielearn/src/migrations/20230922173839_pl_courses__created_at__set_not_null.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE pl_courses
+ALTER COLUMN created_at
+SET NOT NULL;
```

Comment:
I didn't do the fancy multi-stage constraint change because this `pl_courses` table should always be small enough that we can do this synchronously - I think that's true?

Reply:
Yep, sounds fine to me.